### ISSUE TITLE
[bug]: Progress Bar accessibility missing attributes 

### ISSUE THREAD

**Issue by sukh-allwyn** [shadcn-ui/ui#5770]
### Describe the bug

value of the progress bar needs to passed down here. Like this:  `<ProgressPrimitive.Root
      value={value}`
  This will ensure the accessibility values aria-valuenow and aria-valuetext are applied. 
      
  

### Affected component/components

Progress

### How to reproduce

View the HTML element when the progress bar has progress value. 

### System Info

```bash
Browser
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by antoinefamibelle** [comment]
Hi,
@sukh-allwyn I would like to work on this issue can you assign it to me please ?
Thank you
Best regards,

**Comment by sukh-allwyn** [comment]
Seems like I am unable to assign it to anyone.

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
